.github/workflows/main.yml

```diff
@@ -114,4 +114,4 @@ jobs:
     - uses: actions/checkout@v4
     - uses: dtolnay/rust-toolchain@stable
     - name: Run Clippy
-      run: cargo clippy --all-targets --all-features -- -D warnings
+      run: cargo clippy --all-targets --all-features -- --allow clippy::needless-lifetimes -D warnings
```

Comment:
I'd recommend adding `#![allow(clippy::needless_lifetimes)]` to the `lib.rs` file instead.